x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/code_block.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+
+/**
+ * The <EuiCode /> component expect the children provided to be a string (html).
+ * This component allows both string and JSX element
+ *
+ * TODO: Open PR on eui repo to allow both string and React.Node to be passed as children of <EuiCode />
```

Comment:
Do we have an issue open somewhere for this?

Reply:
No, it is still a `TODO`. Going through the `TODOs` might be a good thing to do during the extra week at the end of the cycle.